<launch>
    <node name="command_receiver" pkg="auv" type="command_receiver.py" args="" />
    <node name="thruster_converter" pkg="auv" type="vector_trajectory_converter.py" args="" />
    <node name="filtering" pkg="auv" type="kalman_filter" />
    <node name="control_loop" pkg="auv" type="control_loop_pi.py" />
</launch>